.circleci/config.yml

```diff
@@ -288,4 +340,29 @@ workflows:
               only: develop
             tags:
               only: /^[0-9]+\.[0-9]+\.[0-9]+(-rc\.[0-9]+)?$/
+      - image-build:
+          requires:
+            - deploy
+          filters:
+            branches:
+              only: develop
+            tags:
+              only: /^[0-9]+\.[0-9]+\.[0-9]+(-rc\.[0-9]+)?$/
```

Comment:
We don't need this for this particular one right?  Since only develop?

Reply:
that's because [how](https://circleci.com/docs/2.0/configuration-reference/#tags) circleci handles it, which is weird:

> For a tag push unaffected by any filters, CircleCI skips the job.


this is just like the `deploy` step btw